ch.threema.threema-desktop.yml

```diff
@@ -0,0 +1,72 @@
+app-id: ch.threema.threema-desktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    DEV_ENV: development
```

Comment:
Is DEV_ENV set on purpose here? Shouldn't it point to production or something similar?

Reply:
From what I can tell it doesn't matter on Linux, on macOS something different than `development` triggers a signature/notarize process.  FWIW, Arch and other packagers also set this to `development`.